### ISSUE TITLE
Support homebrew-cask?

### ISSUE THREAD

**Issue by nogweii** [rwoeber/homebrew-update-notifier#5]
Should be as simple as calling `brew cask outdated` and appending the results to a list. It's output is smart, too -- it'll be more verbose if it's executed interactively but in a scripting context it's just a space separated list of out-of-date casks.

**Comment by frdmn** [comment]
Good idea, haven't thought about that yet to be honest. Will add support for cask soon 👍

**Comment by nogweii** [comment]
There​ are a couple of complications that may inform how you set up the notification text. `brew upgrade` doesn't pull in casks, and there is a class of casks that don't have a version policy, just always "latest" (pass `--greedy` to outdated)
